src/docker/Dockerfile

```diff
@@ -52,43 +52,31 @@ RUN apt-get update && apt-get install -y \
     automake autoconf
 
 # Git 2.0
-RUN apt-get install -y software-properties-common python-software-properties
+RUN apt-get update && apt-get install -y \
```

Comment:
@mkordas What exactly do you mean? :D I don't have so much experience with Docker and I don't know what this project needs exactly - I only removed what seemed clearly not needed to me. So just suggest changes if you know for sure they are ok :D